src/Shared/Data.php

```diff
@@ -679,14 +691,22 @@ public function isSubscription($orderId)
         return apply_filters($this->pluginId . '_is_subscription_payment', $isSubscription, $orderId);
     }
 
-    public function getAllAvailablePaymentMethods($use_cache = true)
+    public function getAllAvailablePaymentMethods($use_cache = true, $filters = [])
```

Comment:
Maybe I overlooked it, but it doesn't look like the parameter `$filters `is needed. It is never passed when the method is called.

Reply:
In one of the tries that I was doing I was passing filters here, but then was not needed, I removed it now.